roles/system/tasks/main.yml

```diff
@@ -54,14 +37,10 @@
   when: ansible_lsb.id == "Debian"
   tags: ['master','update']
 
-- name: Run apt-get update  
-  shell: apt-get update ; echo
-  register: result
-  tags: ['master','custom','update']
-
-- name: Fix broken package
```

Comment:
I have the feeling this has been removed (unless it has been moved somewhere else). It should stay as some time dpkg command get corrupted